packages/terra-clinical-result/tests/wdio/result-name-header-cell-spec.js

```diff
@@ -1,25 +1,19 @@
 Terra.describeViewports('ResultNameHeaderCell', ['medium'], () => {
-  describe('default', () => {
-    before(() => {
-      browser.url('/#/raw/tests/terra-clinical-result/clinical-result/result-name-header-cell/result-name-header-cell');
-    });
+  it('displays a default ResultNameHeaderCell', () => {
+    browser.url('/#/raw/tests/terra-clinical-result/clinical-result/result-name-header-cell/result-name-header-cell');
 
-    Terra.it.validatesElement();
+    Terra.validates.element('default');
   });
 
-  describe('paddingStyles', () => {
-    before(() => {
-      browser.url('/#/raw/tests/terra-clinical-result/clinical-result/result-name-header-cell/padding-style-result-name-header-cell');
-    });
+  it('displays a ResultNameHeaderCell with paddingStyles', () => {
+    browser.url('/#/raw/tests/terra-clinical-result/clinical-result/result-name-header-cell/padding-style-result-name-header-cell');
 
-    Terra.it.validatesElement();
+    Terra.validates.element('wth paddingStyles');
```

Comment:
```suggestion
    Terra.validates.element('with paddingStyles');
```

Reply:
unless you meant "what the heck paddingStyles"

Reply:
good catch